src/migrations/20220808084524-add-group-permissions.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        ALTER TABLE group_user DROP COLUMN IF EXISTS role;
+        `,
+        cb,
+    );
+};
+
+exports.down = function (db, cb) {
+    db.runSql(
+        `
+            ALTER TABLE group_user ADD COLUMN role text check(role in ('Owner', 'Member')) default 'Member';
```

Comment:
I'm not convinced this down migration is ideal, since this needs to be lossy, but I think it's better to have a default here so Unleash carries on working, even if this will strip ownership